src/Stratis.Features.FederatedPeg.Tests/CrossChainTransferStoreTests.cs

```diff
@@ -189,6 +190,57 @@ public async Task StoringDepositsWhenWalletBalanceSufficientSucceedsWithDetermin
             }
         }
 
+
+        /// <summary>
+        /// Recording deposits when the wallet UTXOs are sufficient succeeds with deterministic transactions.
+        /// </summary>
+        [Fact]
```

Comment:
This summary is from a different test I guess?

Nice test though

Reply:
Fixed.